www/CONTRIBUTING.md

```diff
@@ -1,29 +1,110 @@
-  ```
-* Run the development server :
-  ```powershell
-  hugo server
-  ```
-* And go to http://localhost:1313/ to see the effect of your changes in livereload
\ No newline at end of file
+Contribution au site web Ergo‑L
+================================================================================
+
+
+Pré-requis : Hugo + Pandoc
+--------------------------------------------------------------------------------
+
+### Windows
+
+Installer [Hugo][] et [Pandoc][] depuis un émulateur de terminal, par exemple
+Windows PowerShell :
+
+```powershell
+winget install Hugo.Hugo.Extended
+winget install --source winget --exact --id JohnMacFarlane.Pandoc
+```
+
+Attention : Hugo ne fonctionne pas depuis le terminal Windows PowerShell qui est
+pré-installé.
+
+Hugo requiert un terminal WSL, Git Bash ou [PowerShell][] tout court, [qui n’est
+pas la même application que Windows PowerShell][WindowsPS], bien que le nom soit
+qussiment le même et que les deux applications sont maintenues par Microsoft.
+PowerShell peut s’installer depuis Windows PowerShell :
+
+```powershell
+winget install --id Microsoft.Powershell --source winget
+```
+
+[Hugo]:       https://gohugo.io/installation/windows/
+[Pandoc]:     https://pandoc.org/installing.html#windows
+[PowerShell]: https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.4
+[WindowsPS]:  https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.3
+
+
+Serveur de développement
+--------------------------------------------------------------------------------
+
+On lance le serveur de développement via un émulateur de terminal depuis ce
+répertoire `www` :
+
+```bash
+cd www
+hugo serve
+```
+
+Le site est alors visible sur http://localhost:1313/
+
+Le site est mis à jour à chaque modification de fichier (live-reload).
+
+
+
+Rédaction
+--------------------------------------------------------------------------------
+
+### Markdown
+
+Les pages sont écrites en Pandoc Markdown chaque fois que c’est possible.
+
+Les lignes sont limitées à 80 caractères. Des exceptions peuvent être admises
+au cas par cas.
+
+Les titres de niveau 1 et 2 sont marqués par un soulignement de 80 signes `=` et
+`-` respectivement. Les titres de niveau supérieur sont indiqués par des `#`,
+mais on évitera d’utiliser les niveaux supérieurs à 3.
+
+Pour les liens, on privilégie les URLs dans une référence de bas de page plutôt
+que dans le corps de texte, afin de faciliter la lecture.
+
+### Orthographe
+
+On suit l’orthographe de la réforme de 1990.
+
+Par souci d’inclusivité, on privilégie les formules épicènes. Les formes
+condensées avec le point médian sont tolérées mais gagnent à être évitées.
+L’accord de proximité est encouragé.
+
+### Typographie
+
+On utilise l’insécable fine avant toutes les ponctuations hautes (`?!:;`) et
+entre les guillemets (`« »`).
+
+Les incises sont délimitées par des tirets cadratins.
```

Comment:
Une règle, vraiment ? Appliquée à coups de `sed` ? Pourquoi pas ?, mais c’est une touche d’archaïsme ou de préciosité qui peut (d)étonner un peu dans un contexte technique. À trancher, en cherchant en tout cas la cohérence au niveau de chaque article.

Reply:
Là encore, j’avoue avoir appliqué une (vive) préférence personnelle. Je me rangerai à l’avis qualifié de la communauté, bien évidemment.

La règle que j’ai trouvée est sur Wikipedia : https://fr.wikipedia.org/wiki/Tiret

> Il sert également à encadrer les [propositions incises](https://fr.wikipedia.org/wiki/Proposition_incise) (et exceptionnellement les éléments incidents) dans la fonction de quasi-parenthèse. Il lui est souvent conféré une valeur littéraire par rapport à la parenthèse[2](https://fr.wikipedia.org/wiki/Tiret#cite_note-2).

Reply:
Allez, hop, il faut se faire plaisir parfois : tirets cadratins-longs-à-l’ancienne *it is* ! Ça donne l’occasion de les pratiquer. Le résultat peut être ostentatoire avec certaines polices, mais Ergol n’a rien à cacher…